node/impl/full/eth.go

```diff
@@ -689,6 +689,9 @@ func (a *EthModule) EthFeeHistory(ctx context.Context, p jsonrpc.RawParams) (eth
 	}
 	rewardPercentiles := make([]float64, 0)
 	if params.RewardPercentiles != nil {
+		if len(*params.RewardPercentiles) >= 100 {
```

Comment:
```suggestion
		if len(*params.RewardPercentiles) > 100 {
```

(to match the error description)

Reply:
I'd also make the limit a constant at the top of the file.

Reply:
Done.